lib/publify/api.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+
+module Publify
+  class API
+    def self.latest_links(limit)
+      uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+      response = Net::HTTP.get uri
```

Comment:
Also, do we know what the timeout is here? It should probably be short to avoid a stalled publify (or heroku issue) causing cms/frontend to be unacceptably sluggish.